custom_components/waste_collection_schedule/waste_collection_schedule/source/kalundborg_dk.py

```diff
@@ -0,0 +1,93 @@
+from datetime import datetime
+
+import requests
+from waste_collection_schedule import Collection  # type: ignore[attr-defined]
+
+TITLE = "Kalundborg Kommune"
+DESCRIPTION = "Source for Kalundborg Kommune waste collection schedule"
+URL = "https://kalundborg.dk"
+TEST_CASES = {
+    "TestCase1": {
+        "id": "00006ac8-0002-0001-4164-647265737320",
+    },
+    "TestCase2": {
+        "id": "00008d18-0002-0001-4164-647265737320",
+    },
+    "TestCase3": {
+        "id": "00006643-0002-0001-4164-647265737320",
+    },
+}
+
```

Comment:
adding a description on how to find the id that shows up in the GUI would probably be nice

```py
HOW_TO_GET_ARGUMENTS_DESCRIPTION = {
    "en" : "SOME TEXT"
}
```